### PR TITLE
Start documentation chapter for plugin tests

### PR DESCRIPTION
**Pull request by SergeoLacruz**
Lets start from here. I will continue as soon as I have something working...

### CODE REVIEW

**Comment by SergeoLacruz** [comment]
Format looks ugly...
Will fix tomorrow
Good night :-)

**Comment by SergeoLacruz** [comment]
Looks better now.

**Comment by SergeoLacruz** [comment]
Thanks. I added your comments. I think we can publish it for the moment and add details as soon as we have them.

File: docs/docs/extend/plugins/test.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+Title: Unit Tests
+---
+
+## Unit Tests
+For complicated plugins it makes sense to add unit tests to your code. InvenTree
```
Comment (wolflu05) [inline docs/docs/extend/plugins/test.md:6]: Maybe add something like "to ensure that plugins work correctly and are compatible with future versions too. You can run these tests as part of your ci against the current stable and latest tag to get notified when something breaks before it gets released as part of stable"???

**Review by wolflu05 — APPROVED** [review]

**Comment by SchrodingersGat** [comment]
In the "do some work here" I think it would be a good idea to have some examples of what testing might actually be performed

**Comment by SergeoLacruz** [comment]
Definitely...
But that's the next chapter :-)

**Pull request closed without merging**